man/nvim.1

```diff
@@ -0,0 +1,375 @@
+.Dd May 7, 2015
+.Dt NVIM 1
+.Os Neovim
+.Sh NAME
+.Nm nvim
+.Nd edit text
+.Sh SYNOPSIS
+.Nm
+.Op Ar options
+.Op Ar file ...
+.Nm
+.Op Ar options
+.Fl
+.Nm
+.Op Ar options
+.Fl t Ar tag
+.Nm
+.Op Ar options
+.Fl q Ar errorfile
+.Sh DESCRIPTION
+.Nm
+is a text editor based on
+.Xr vim 1 .
+To enter commands in
+.Nm ,
+type a colon
+.Sy ( \(dq:\(dq ) ,
+which is also used in this manual to denote commands.
+For more information, consult the on-line help system with the
+.Ic :help
+command.
+.Bl -tag -width Fl
+.It Ar file ...
+File(s) to edit.
+If none are specified, open an empty buffer.
+If multiple files are specified, open one buffer for each file.
+To switch between buffers, use the
+.Ic :next
+and
+.Ic :previous
+commands.
+.It Fl
+Read text from standard input until EOF, then open a buffer with that text.
+Commands are read from standard error, which should be a terminal.
+.It Fl t Ar tag
+The file to edit and the initial cursor position depends on a
+tag, a sort of goto label.
+.Ar tag
+is looked up in the tags file, the associated file becomes the current
+file and the associated command is executed.
+If
+.Ar tag
+is a function name, the file containing that function is opened
+with the cursor positioned at the start of the function.
+See
+.Ic :help tag-commands .
+.It Fl q Ar errorfile
+QuickFix mode; display the first error in
+.Ar errorfile .
+If
+.Ar errorfile
+is omitted, the filename is obtained from the
+.Li 'errorfile'
+option (defaults to
+.Pa errors.err ) .
+Further errors can be jumped to with the
+.Ic :cnext
+command.
+See
+.Ic :help quickfix .
+.It There are a number of other options:
+.It Fl -
+Interpret all further arguments as files.
+Can be used to edit files starting with a hyphen.
+.It Fl -literal
+Interpret filenames literally, that is do not expand wildcards.
+Has no effect on UNIX-like systems, where the shell expands wildcards.
+.It Fl e
+Ex mode.
```

Comment:
:+1: I'll update the rest of the "mode" flags, but keep the `--help` output the same for brevity.